sw/device/tests/dif/dif_plic_unittest.cc

```diff
@@ -20,7 +20,7 @@ using testing::Test;
 
 // If either of these static assertions fail, then the unit-tests for related
 // API should be revisited.
-static_assert(RV_PLIC_PARAM_NUMSRC == 122,
+static_assert(RV_PLIC_PARAM_NUMSRC == 172,
```

Comment:
Holy Cow.... I knew this was coming, but it's quite shocking to see this bump in the number of interrupts!

Reply:
yeah, so you can see why we're going to start asking people to consolidate interrupts now :)
I'll be commenting on that for spi host Martin!

Reply:
Well, for SPI host I tried to go super lean, with just two: "Event" and "Error".  (Not sure if you've had a chance to look yet)